corehq/apps/accounting/tests/generator.py

```diff
@@ -155,9 +155,15 @@ def arbitrary_domain_and_subscriber():
 
 
 @unit_testing_only
-def arbitrary_commcare_user(domain, is_active=True):
+def arbitrary_commcare_user(domain, is_active=True, is_webuser=False):
```

Comment:
i feel like this method can be renamed to be a little more clear since it's creating web users and commcare users now

Reply:
https://github.com/dimagi/commcare-hq/pull/32747/commits/1b41679e73406846e3d54702e83bba708adf535a